src/main/drivers/max7456.c

```diff
@@ -502,10 +541,10 @@ void max7456DrawScreenPartial(void)
             if (buff_len > 0)
                 max7456SendDma(spiBuff, NULL, buff_len);
             #else
-            ENABLE_MAX7456;
+            ENABLE_MAX7456();
             for (k=0; k < buff_len; k++)
                 spiTransferByte(MAX7456_SPI_INSTANCE, spiBuff[k]);
```

Comment:
`spiTransfer` should be more efficient for bulk transfers

Reply:
We can also optimize spiTransfer to use 16-bit transfer where possible, but that's a scope of another PR

Reply:
It seems DMA is enabled depending on the target configuration. Only `SPRACINGF3NEO` seems to use it (both for TX and RX). Do all boards which might define `MAX7456_DMA_CHANNEL_TX` also allow sending the MAX7456 commands via SPI? Also note that we'll be sending 30 bytes at most (36 if we end up using HW blinking).

Reply:
In many cases MAX shares the bus with gyro which makes using DMA tricky at best. As long as the code does check for `dmaTransactionInProgress` we can mix DMA'ed and polled access.